registration/include/pcl/registration/impl/transformation_estimation_svd_scale.hpp

```diff
@@ -77,7 +77,6 @@ pcl::registration::TransformationEstimationSVDScale<PointSource, PointTarget, Sc
 
   Eigen::Matrix<Scalar, Eigen::Dynamic, Eigen::Dynamic> src_ = R4 * cloud_src_demean;
   
-  float scale2;
   double sum_ss = 0.0f, sum_tt = 0.0f, sum_tt_ = 0.0f;
```

Comment:
`sum_tt` also appears to be unused. Maybe delete L87-89, replace `sum_tt_` by `sum_tt` as the variable name and remove declaration of `sum_tt_`

Reply:
Much better. Thanks!